istioctl/cmd/workload.go

```diff
@@ -217,6 +218,7 @@ Configure requires either the WorkloadGroup artifact path or its location on the
 	configureCmd.PersistentFlags().StringVar(&clusterID, "clusterID", "Kubernetes", "The ID used to identify the cluster")
 	configureCmd.PersistentFlags().Int64Var(&tokenDuration, "tokenDuration", 3600, "The token duration in seconds (default: 1 hour)")
 	configureCmd.PersistentFlags().StringVar(&ingressIP, "ingressIP", "", "IP address of the ingress gateway")
+	configureCmd.PersistentFlags().BoolVar(&autoRegister, "autoregister", false, "If set, a WorkloadEntry will be created upon connection to istiod (if enabled in pilot).")
```

Comment:
Is it possible to add an example so if user is looking for how to use this flag they will just run `-h`.

Reply:
Hmm.. not sure exactly; they don't need to supply anything additional if using this flag. The idea is they just skip steps later in whatever doc they're following. The docs are WIP right now.